fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -46,6 +46,22 @@ private FHIRRegistry() {
         providers = new CopyOnWriteArrayList<>(loadProviders());
     }
 
+    /**
+     * Get a list of registry resources that have the given resource type.
+     *
+     * @param resourceType
+     *     the resource type
+     * @return
+     *     a list of registry resources that have the given resource type
+     */
+    public List<FHIRRegistryResource> getRegistryResources(Class<? extends Resource> resourceType) {
```

Comment:
how does it relate to `public <T extends Resource> Collection<T> getResources(Class<T> resourceType)` ?

Reply:
ah, I see it now.  this one returns RegistryResources (metadata) and the other one returns the actual resources...wonder if its worth adding any explanation about when to use this one vs the other (i.e. use this one to get the resource metadata and not the resource contents)

Reply:
One method gets FHIR resources the other gets Registry resources.

Reply:
maybe move it next to those other ones since its pretty close (at least in name)?

Reply:
Sure